source/administration/changelog.md

```diff
@@ -4,9 +4,269 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+- Added support for experimental gossip protocol, where the server will attempt to communicate via the gossip protocol over the gossip port.
+
+### Improvements
+
+#### Web User Interface
+- Adjusted post spacing to be consistent across Markdown formatting, replies and consecutive posts.
+- On hover colour for pin and channel member icons now consistent with flag and recent mentions icons.
+- Emojis are now vertically aligned in post view.
+- Channel name, header and purpose now update in real time for all users.
+- For reply threads in the center channel, the "Commented on" phrase now respects the teammate name display setting.
+- Code block language tag is no longer selectable making it easier to copy the code.
+- Aligned the search box with right-hand side reply thread.
+- New user profile pictures now update for other users upon refresh.
+- Improved rendering of @mention highlighting in message view.
+
+#### Mobile Web
+- Added "Create Team" and "Leave Team" options to the Main Menu.
+- Updated the look of Account Settings pages on mobile.
+- User profile popover no longer gets cropped in the center channel on iOS browser.
+- Link preview image now resizes correctly on iOS browser.
+
+#### Notifications
+- Unread messages and mentions now sync across browser tabs and devices.
+- Improved desktop notifications for webhook attachments.
+
+#### Emoji Picker & Custom Emoji
+- Newly created custom emoji immediately display to all users without requiring a refresh.
+- Improved position of the emoji picker near the top of the channel or the right-hand side comment thread.
+
+#### Keyboard Shortcuts
+- CTRL+SHIFT+K shortcut now toggles the Direct Message dialog open and closed.
+- SHIFT+UP now opens a reply thread for the most recent message posted by a user, skipping system messages.
+
+#### Slash Commands
+- Added the following built-in slash commands:
+  - `/header` command to set the channel header.
+  - `/help` command to open the Mattermost help page in a new browser tab.
+  - `/open` command to switch or join a channel.
+  - `/search` command to search text in messages.
+  - `/settings` command to open the Account Settings dialog.
+- `/invite_people` slash command is now disabled when account creation is set to false.
+- If a message starts with a / but fails to send (either due to timeout or invalid command), the message is put back to the input box.
+
+#### Bulk Import Tool
+- Added support for Direct Message channels and posts to the [bulk import tool](https://docs.mattermost.com/deployment/bulk-loading.html).
+
+#### Authentication
+- User creation via OAuth (GitLab/Google/Office365) properly restricted to accepted domains, [if specified](https://docs.mattermost.com/administration/config-settings.html#restrict-account-creation-to-specified-email-domains).
+- **Invite New Member** dialog validates email addresses against accepted domains, if set.
+
+#### New URL Routes // XXX Need better name and description of benefit
```

Comment:
Whoops yes. They're not API changes, they're URL changes.

Like you can open a DM channel with someone by pasting a URL of the form https://pre-release.mattermost.com/messages/@jason